ui/userguide.html

```diff
@@ -641,12 +641,14 @@
              title="Creative Commons BY 3.0">CC BY 3.0</a>
         </div>
       </div>
+      <!-- Disabling whatsdeployed for now -- https://bugzilla.mozilla.org/show_bug.cgi?id=1208086 -->
+      <!--
       <div class="col-xs-6">
         <a class="midgray pull-right"
            href="https://whatsdeployed.paas.allizom.org/?owner=mozilla&amp;repo=treeherder&amp;name[]=Heroku-prototype&amp;url[]=https://treeherder-heroku.herokuapp.com/revision.txt&amp;name[]=Stage&amp;url[]=https://treeherder.allizom.org/revision.txt&amp;name[]=Prod&amp;url[]=https://treeherder.mozilla.org/revision.txt">What's Deployed?</a>
       </div>
     </div>
-
+    -->
```

Comment:
The browser is likely compensating for the mismatched `<div>`s.

With this PR there are more `<div>`s than `</div>`s.